app/routes/send-endringsmelding.tsx

```diff
@@ -18,6 +18,6 @@ export async function action({ request }: ActionArgs) {
   });
 }
 
-export default function SendEndringsmelding() {
+export default function SendEndringsmeldingSide() {
```

Comment:
Hvorfor trengte denne å endre navn?

Reply:
Fikk feil på import. `Sendendringsmelding()` er også en funkjson som sender endringsmelding i tillegg til et komponent for siden. Ligger et teknisk gjeld kort ute nå på å fikse dette. Vil du at eg skal endre navnet tilbake? 🤩

https://trello.com/c/cMNCnxD1/142-fjerne-gammel-kode-for-gamle-oppsettet